### PR TITLE
Added new feature method: updateByUniqueId

### PR DESCRIPTION
**Pull request by sketchman-nyt**
Allows for a row to be selected by UniqueId and then updated with new data. Leverages the existing getRowByUniqueId and has a precedent in removeByUniqueId.

### CODE REVIEW

**Comment by tm1000** [comment]
:+1:

**Comment by djhvscf** [comment]
This PR must be against develop branch @sketchman-nyt

**Comment by wenzhixin** [comment]
same feature with #1514, consider this one, but please commit to develop branch and help to update to docs, thanks!

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -2193,6 +2193,20 @@
...
     };
 
+    BootstrapTable.prototype.updateByUniqueId = function (params) {
+        var rowId;
+
+        if (!params.hasOwnProperty('id') || !params.hasOwnProperty('row')) {
+            return;
+        }
+
+        rowId = this.options.data.indexOf(this.getRowByUniqueId(params.id));
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: IE8 is not support this method, please use $.inArray instead.

**Comment by sketchman-nyt** [comment]
Thanks for the comments. I've created a new PR to develop branch since I can't update the branch in this one. #1592 is the new branch.

Also, for the docs update, I need to edit that repo, not this one, correct?

**Comment by wenzhixin** [comment]
Yes, you are right, close this one.

**Pull request closed without merging**